components/Navigation.js

```diff
@@ -1,38 +1,80 @@
 import Link from "next/link";
 import styled from "styled-components";
-import IconAdd from "@/public/iconAdd.svg";
-import IconHome from "@/public/iconHome.svg";
+import AddIcon from "@/public/icons/plus.svg";
+import HomeIcon from "@/public/icons/home.svg";
 import { useAtom } from "jotai";
 import { statusUploadAtom } from "@/lib/atom";
+import { useRouter } from "next/router";
+import { useState } from "react";
 
 export default function Navigation() {
   const [statusUpload, setStatusUpload] = useAtom(statusUploadAtom);
+  const router = useRouter();
 
   return (
-    <List>
-      <li>
-        <Link href="/" onClick={() => setStatusUpload("")}>
-          <IconHome />
-        </Link>
-      </li>
-      <li>
-        <Link href="/create-new-project">
-          <IconAdd />
-        </Link>
-      </li>
-    </List>
+    <StyledNavigation>
+      <List>
+        <li>
+          <Link href="/" onClick={() => setStatusUpload("")}>
+            <HomeIconWrapper isFocused={router.pathname}>
+              <HomeIcon />
+            </HomeIconWrapper>
+          </Link>
+        </li>
+        <li>
+          <Link href="/create-new-project">
+            <AddIconWrapper isFocused={router.pathname}>
+              <AddIcon />
+            </AddIconWrapper>
+          </Link>
+        </li>
+      </List>
+    </StyledNavigation>
   );
 }
 
+const HomeIconWrapper = styled.div`
```

Comment:
To use a div inside of an anchor element is not the solution I would prefer. [See next.js documentation on how to style a next/link with styled-components.](https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag) But you can also style the link directly. [See art gallery app navigation.](https://github.com/mbosselmann/art-gallery-app/blob/main/components/Navigation/index.js)